shared/modules/doc/CHANGELOG.md

```diff
@@ -0,0 +1,40 @@
+# Changelog
+
+All notable changes to this module will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.0.1] - 2022-08-21
+- Minimized shared module with #163
```

Comment:
NIT (in here and/or for the future): Please add a hyperlink to the issue like so:

```
- Minimized shared module with [#163](https://github.com/pokt-network/pocket/issues/163)
```

newcomers may not know what 163 means.